bin/preview-deploy/aws.sh

```diff
@@ -15,15 +15,15 @@ function deployPreviewtoEC2() {
   configureAWS
 
   # Inject branch name into user data
-  configureUserData $2
+  configureUserData
```

Comment:
Isn't `configureUserData` still acting on the branch name? Does it not need to be passed in? I AM SO BAD AT BASH.

Reply:
Ah, bad comment! You're right, it is, but it's getting it from the `$BRANCH` local global (I don't know what I'm saying either), so it doesn't have to be passed in as an explicit argument anymore. Going to push an updated comment.